OpenSim/Common/DataTable.h

```diff
@@ -0,0 +1,268 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  DataTable.h                           *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
```

Comment:
The other thing I like about putting my name on the `Authors:` list is that it makes me co-copyright holder with Stanford (because of the previous line). The only practical implication of that as I understand it is that the license terms could not then be changed without my agreement.